src/components/CreateCourse/CreateCourse.jsx

```diff
@@ -227,4 +231,24 @@ const CreateCourse = () => {
 	);
 };
 
-export default CreateCourse;
+const mapStateToProps = (state) => {
+	return {
+		authorsForm: state.authors.authorsForm,
+		authors: state.authors.authors,
+		courses: state.courses.courses,
+		enabledAuthors: state.authors.enabledAuthors,
+	};
+};
+
+const mapDispatchToProps = (dispatch) => {
+	return {
+		getCourses: (data) => dispatch(actionCreators.getCourses(data)),
```

Comment:
having `get` in action creator name is not correct, as your action creator doesn't return anything